samples/projects/application_for_backend/qsharp.json

```diff
@@ -0,0 +1,8 @@
+{
+    "packageType": "exe",
+    "dependencies": {
+        "ContosoBackend": {
+            "path": "../quantum_backend"
```

Comment:
Maybe this should be a git dependency? It'll build then in isolation without cloning the entire "samples" dir.

If you use your own commit as the `ref` parameter, it'll build now. We can update that ref later.

Or, we could leave it as is and assume all consumers will only build this in the context of the entire `samples` dir.

Reply:
Good point, I'll make it a git dependency.

Reply:
Updated.